### PR TITLE
Capstone project review

### PR DESCRIPTION
**Pull request by ShahierNashaat**
Pull request for reviewing capstone project for javascript module using HTML5, CSS3, JavaScript, ES6, Webpack, Jest, and Kanban board.

- Find external API
- Choose assets
- Set up the project with Webpack and Jest
- Add header
- Add footer
- Display list of items on the Homepage
- Display Comments pop up with selected item's details
- Display the number of likes for each item on the Homepage
- Display comments for a given item on the Comments pop-up
- Create feature: add new like
- Create feature: add a new comment
- Add all items counter on the Homepage 
- Add comments counter
- Add tests for items counter
- Add tests for comments counter

### CODE REVIEW

File: src/api-controller.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    return [];
+  });
+
+const postLike = (id) => fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/izzKZgflb43lj8vOIg9b/likes', {
+  method: 'POST',
+  body: JSON.stringify({
+    item_id: id,
+  }),
```
Comment (dasileker) [inline src/api-controller.js]: - [ ] I see you did a great job, but you should have used the `async and await` instead.

ooooo

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ShahierNashaat,** 

### **To Highlight**

- The **PR** is really good.
- The Commits are good and consistent.
- The Readme file is nice.
- The coding is clean.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,59 +1,56 @@
...
+- HTML, CSS, and JavaScript
+- Webpack
+- Jest
+- Github projects
+- Kanban board
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://github.com/ShahierNashaat/javascript-capstone/dist)
```
Comment (carlos-ssh) [inline README.md]: [OPTIONAL]
- Kindly update your demo link version

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hello Team! 👋🏽 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**